lib/spreedly.ex

```diff
@@ -230,6 +230,21 @@ defmodule Spreedly do
     get_request(env, list_transactions_path(), params)
   end
 
+  @doc """
+  Retrieve a list of created gateways in the environment with optional
+  query params specified as a keyword list.
+
+  ## Examples
+
+    list_created_gateways(env)
+    list_created_gateways(env, [order: :desc, since_token: "token"])
+
+  """
+  @spec list_created_gateways(Environment.t, Keyword.t) :: {:ok, any} | {:error, any}
+  def list_created_gateways(env, params \\ []) do
```

Comment:
What would ya'll think of `list_gateways` instead? Does `created` buy us anything?

Reply:
We have another API that *lists supported gateways* https://docs.spreedly.com/reference/api/v1/#list-supported-gateways.

I think using just `list_gateways` could make it ambiguous.

Reply:
@duff I had pondered changing it to `list_gateways` also. Using `list_created_gateways` matches the documentation and avoids the confusion of listing a gateway's for an environment vs. list supported gateways (though that could be mitigated through module docs).

Although, we could let the function arity drive the request. Here are some additional naming options, any strong opinions?
 
```
# List available gateway options
def list_gateways()

# List gateways by environment
def list_gateways(env, params \\ [])
```

Or 

```
# List available gateway options
def list_gateway_options()

# List gateways by envi
def list_gateways(env, params \\ [])
```

Or 

```
# List available gateway options
def list_supported_gateways()

# List gateways by envi
def list_created_gateways(env, params \\ [])
```

Reply:
That last one works for me. Whatch ya'll think?

Reply:
👍

Reply:
👍